app/views/posts/index.html.erb

```diff
@@ -1,9 +1,31 @@
-<h1>All Posts for <%= @user.name %></h1>
+<div class="main-container">
+  <div class="users-wrapper flex">
+    <div class="img-container">
+      <%= image_tag @user.photo, :size => "70x80", alt: "user photo" %>
+    </div>
```

Comment:
[OPTIONAL] It would be best if you fixed the image display so that user profile images are displayed properly. Since you haven't implemented a mechanism for adding user images you can create a functionality that displays a default image if a user image doesn't exist in the database.

![image](https://github.com/Balikuddembe/blog-app-rails/assets/38088818/c1959851-72cb-4004-b664-be8613f231fc)

Reply:
Thank you @KDlamini ,all the review comments have been resolved.